components/image-upload.tsx

```
@@ -165,12 +183,18 @@
+                url.split(".").pop()!.toLowerCase(),
+              );
 
             return (
               <div key={index} className="image-preview group">
                 {isVideo ? (
-                  <video src={url} className="w-full h-32 object-cover rounded-lg" controls />
+                  <video
+                    src={url}
+                    className="w-full h-32 object-cover rounded-lg"
+                    controls
+                  />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)